packages/cli/src/runDeploy.ts

```diff
@@ -18,6 +18,8 @@ import { build } from "./build";
 import { kmsKeyToAccount } from "@latticexyz/common/kms";
 import { configToModules } from "./deploy/configToModules";
 
+export const localChains = [1337, 31337];
```

Comment:
Given all of these new tools that lift logic from `deploy`, we need to restructure as suggested in https://github.com/latticexyz/mud/pull/2849#discussion_r1609704642